### PR TITLE
Use `datacenter` field for the `IpInfo` signal instead of the wrong `dataCenter`, deprecate `IPLocation`

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

File: docs/IpInfoResultV4.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Name | Type | Description | Notes
 **address** | **str** |  | [optional] 
 **geolocation** | [**IPLocation**](IPLocation.md) |  | [optional] 
 **asn** | [**ASN**](ASN.md) |  | [optional] 
+**datacenter** | [**DataCenterInfo**](DataCenterInfo.md) |  | [optional] 
 **data_center** | [**DataCenter**](DataCenter.md) |  | [optional]
```
Comment (JuroUhlar) [inline docs/IpInfoResultV4.md:10]: @ilfa is there an easy way to add "Deprecated" to the description or notes here? Right now you have to click through stuff to see which is the correct one. 

```suggestion
**data_center** | [**DataCenter**](DataCenter.md) | Deprecated | [optional] 
```

**Comment by JuroUhlar** [comment]
@ilfa  Tested locally and can confirm that the data center info is now available in `datacenter` with `data_center` being `None`, same as before. I left one suggestion regarding the generated docs, not sure if feasible. If not, consider the PR approved. :) 

image

**Review by JuroUhlar — COMMENTED** [review]
Left one suggestion

**Review by TheUnderScorer — APPROVED** [review]

**Review by JuroUhlar — APPROVED** [review]
Tested locally, works as expected

**Pull request merged**